tests/test_configuration.py

```diff
@@ -1,10 +1,18 @@
 import json
-from unittest.mock import ANY, Mock
+from typing import List
+from unittest.mock import ANY, Mock, call
 
 from django.test import override_settings
-
-from ariadne.types import ExtensionSync
-
+from graphql.language import FieldNode
+from graphql.pyutils import Path
+
+try:
+    from ariadne.types import ExtensionSync
+except ImportError:
+    # From ariadne 0.20 Extension supports both sync and async contexts
+    # https://github.com/mirumee/ariadne/blob/main/CHANGELOG.md#020-2023-06-21
+    from ariadne.types import Extension as ExtensionSync
+from graphql import ExecutionContext, GraphQLBoolean, GraphQLResolveInfo, GraphQLScalarType
```

Comment:
```suggestion
from graphql import ExecutionContext, GraphQLBoolean, GraphQLResolveInfo, GraphQLScalarType
```

Please format code with black.